learning_resources/serializers.py

```diff
@@ -103,29 +136,208 @@ class LearningResourceBaseSerializer(serializers.ModelSerializer):
     audience = serializers.ReadOnlyField()
     certification = serializers.ReadOnlyField()
     prices = serializers.ReadOnlyField()
+    topics = WriteableSerializerMethodField()
```

Comment:
Perhaps premature since this is still a PR, but I opened #54 about `topics` here.